.github/workflows/ci.yml

```diff
@@ -0,0 +1,119 @@
+  MAVEN_INSTALL_CMD: "mvn clean install -B"
+
+jobs:
+  pre_commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: Alfresco/alfresco-build-tools/.github/actions/pre-commit@v1.35.0
+
+  build:
+    name: "Build application"
+    runs-on: ubuntu-latest
+    needs: [pre_commit]
+    if: "!contains(github.event.head_commit.message, '[skip tests]')"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.35.0
+      - name: "Build"
+        run: mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
+      - name: "Verify"
+        run: mvn $MAVEN_CLI_OPTS verify -Dlogging.root.level=off -Dspring.main.banner-mode=off
+
+  tests:
+    name: ${{ matrix.name }}
+    runs-on: ubuntu-latest
+    needs: [build]
+    if: "!contains(github.event.head_commit.message, '[skip tests]')"
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: "current version Java 17"
+            java-version: 17
+          - name: "current version Java 11"
+            java-version: 11
+          - name: "7.3 Enterprise Java 17"
+            java-version: 17
+            suite: -Penterprise-73-tests
+          - name: "7.3 Community Java 17"
+            java-version: 17
+            suite: -Pcommunity-73-tests
+          - name: "7.3 Enterprise Java 11"
+            java-version: 11
+            suite: -Penterprise-73-tests
+          - name: "7.3 Community Java 11"
+            java-version: 11
+            suite: -Pcommunity-73-tests
+          - name: "7.2 Enterprise"
+            java-version: 11
+            suite: -Penterprise-72-tests
+          - name: "7.2 Community"
+            java-version: 11
+            suite: -Pcommunity-72-tests
+          - name: "7.1 Enterprise"
+            java-version: 11
+            suite: -Penterprise-71-tests
+          - name: "7.1 Community"
+            java-version: 11
+            suite: -Pcommunity-71-tests
+          - name: "7.0 Enterprise"
+            java-version: 11
+            suite: -Penterprise-70-tests
+          - name: "7.0 Community"
+            java-version: 11
+            suite: -Pcommunity-70-tests
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v1.35.0
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.35.0
+        with:
+          java-version: ${{ matrix.java-version }}
+      - name: "Login to Docker Hub"
+        uses: docker/login-action@v2.1.0
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: "Login to Quay.io"
+        uses: docker/login-action@v2.1.0
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+      - name: "Build"
+        run: ${MAVEN_INSTALL_CMD} ${{ matrix.suite }}
```

Comment:
Let's replace this `MAVEN_INSTALL_CMD` with the actual command. It was useful on Travis to avoid repetition, but here we already avoided the repetition via the matrix approach.